packages/@lwc/ssr-compiler/src/compile-template/transformers/component/slotted-content.ts

```diff
@@ -40,17 +40,17 @@ const bGenerateSlottedContent = esTemplateWithYield`
             // Avoid creating the object unnecessarily
             : null;
 
-        const lightSlottedContent = ${/* hasLightSlottedContent */ is.literal} 
+        const lightSlottedContentChildren = ${/* hasLightSlottedContent */ is.literal} 
```

Comment:
Open to changing the name on this but essentially this fixes the slot forwarding.

When a light slot is slotted into a shadow slot, the `lightSlottedContent` references the one in the closure rather than the the one passed to the template function.

Reply:
We were looking at other slot stuff yesterday, and I thought it was inconvenient using the same name for things in different places. Interesting that it was also causing bugs!

Reply:
`children` feels like an array of nodes to me, this is a map of slot name to functions that create nodes, right? `lightSlotNameToContentGeneratorMap` would be accurate but verbose, maybe just `lightSlottedContentMap`?

Reply:
:+1: for `lightSlottedContentMap`. It distinguishes it from `shadowSlottedContent` which is not a map.